src/components/Localization2DViewer.tsx

```diff
@@ -27,14 +28,13 @@ export default function Localization2DViewer(props:
   useEffect(() => {
     const loadMap = async () => {
       let segments = [];
-      if (mapName !== "") {
-        const res = await fetch(config.mapUrl(mapName));
-        if (res.ok) {
-          try {
-            segments = await res.json();
-          } catch (err) {
-            console.error(`Bad map data for "${mapName}"`, err);
-          }
+      const mapName = rc.mapName!;
```

Comment:
I would go for a default empty value maybe, unless you know it won't be null and the type checker just sucks